broker/metrics.go

```diff
@@ -82,6 +82,10 @@ func (m *Metrics) UpdateCountryStats(addr string) {
 	var country string
 	var ok bool
 
+	if m.countryStats.addrs[addr] {
```

Comment:
Go suggests the use of two-value assignments to test for the existence of a key, but I think relying on the zero value of a `bool` should also be fine: https://blog.golang.org/go-maps-in-action#TOC_3.